src/main/java/sirius/biz/protocol/JournalData.java

```diff
@@ -85,10 +85,7 @@ protected void onSave() {
      * @return a stream of all journaled properties
      */
     public Stream<Property> fetchJournaledProperties() {
-        return owner.getDescriptor()
-                    .getProperties()
-                    .stream()
-                    .filter(p -> p.getAnnotation(NoJournal.class).isEmpty());
+        return owner.getDescriptor().getProperties().stream().filter(p -> p.getAnnotation(NoJournal.class).isEmpty());
```

Comment:
**Optional:** When you are improving it can you also replace `p` with `property`?